.changes/window-state-map-label.md

```diff
@@ -0,0 +1,5 @@
+---
+"window-state": patch
+---
+
+Add `map_label` option to transform the window label before using it internally, based on [amrbashir's suggestion](https://github.com/tauri-apps/plugins-workspace/pull/531#pullrequestreview-1566282300).
```

Comment:
```suggestion
Add `Builder::map_label` option to transform the window label when saving the window state, this could be used to group different windows to use the same state.
```